packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableClaimableRow.tsx

```diff
@@ -351,7 +351,7 @@ export function TransactionsTableClaimableRow({
     <div
       data-testid={`withdrawal-row-${tx.txId}`}
       className={twMerge(
-        'relative grid h-full grid-cols-[150px_250px_140px_290px_140px] border-b border-dark text-sm text-dark',
+        'relative grid h-full grid-cols-[150px_250px_140px_320px_110px] border-b border-dark text-sm text-dark',
```

Comment:
Gotta expand the tx id column, otherwise the network name is too long. Everything should still work and look the same